chapters/de/slides/chapter4_02_running-training.md

```diff
@@ -0,0 +1,269 @@
+---
+type: slides
+---
+
+# Konfigurieren und Ausführen des Trainings
+
+Notes: Lass uns jetzt, wo du gelernt hast, wie du Trainingsdaten erstellst,
+einen Blick darauf werfen, wie du deine Pipeline trainierst und das Training
+konfigurierst. In dieser Lektion wirst du alles über spaCys
+Trainings-Konfigurations-System lernen, wie du deine eigene Trainings-Config
```

Comment:
lernen: Wie ... generierst; wie ...nutzt; und wie

Reply:
I don't typically use semicolons anywhere in German so I'm worried that this makes things too inconsistent.